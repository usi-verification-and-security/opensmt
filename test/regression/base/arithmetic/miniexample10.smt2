(set-logic QF_LIA)
(declare-fun x () Int)
(define-fun uninterp_mul ((a Int) (b Int)) Int (* 2 a b))
(assert (= (uninterp_mul 5 x) 10))
(check-sat)
(get-model)
(exit)