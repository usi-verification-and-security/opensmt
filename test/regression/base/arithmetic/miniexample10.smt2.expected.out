sat
(
  (define-fun x () Int
    1)
)
